sickbeard/webserve.py

```diff
@@ -3590,7 +3636,7 @@ def massEdit(self, toEdit=None):
         air_by_date_value = last_air_by_date if air_by_date_all_same else None
         root_dir_list = root_dir_list
 
-        return t.render(showList=toEdit, showNames=showNames, default_ep_status_value=default_ep_status_value,
+        return t.render(showList='|'.join(showIDs), showNames=showNames, default_ep_status_value=default_ep_status_value,
```

Comment:
This is where I pretty much stopped removing all the csv crap. I could continue of course and also remove this. But then were do you stop?